fusillade/directory/resource.py

```diff
@@ -319,6 +319,7 @@ def get_policy(self, policy_name):
             attrs = dict([(attr['Key']['Name'], attr['Value'].popitem()[1]) for attr in resp['Attributes']])
         except cd_client.exceptions.ResourceNotFoundException:
             raise FusilladeNotFoundException(f"{self.get_policy_path(policy_name)} does not exist.")
+        attrs['policy_document'] = json.loads(attrs['policy_document'])
```

Comment:
Just an observation, related to changing the policy document format from dictionary to JSON string. AWS also has some inconsistencies in how boto returns policy documents (some API endpoints return policies as a dict, others as a JSON string). May be useful to clarify somewhere (I don't really know where...?) that policy documents are stored as stringified JSON.

Reply:
The openapi schema documents the policy_document format. In a future PR I will clean up the openapi spec to make sure everything is consistent.